gcloud/monitoring/timeseries.py

```diff
@@ -127,6 +127,7 @@ class Point(collections.namedtuple('Point', 'end_time start_time value')):
     :type start_time: string or None
     :param start_time: An optional start time in RFC3339 UTC "Zulu" format.
 
+    :type value: int
```

Comment:
pdoc fails if there isn't a type defined for a param, which is how I found it.

I'm game for an `object` for now. Thanks!